### DISCUSSION TITLE
Markdown mode

### DISCUSSION THREAD

**Discussion by benjamonnguyen** [mealie-recipes/mealie#3000] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

The GUI and JSON modes of creating recipes do not fit my workflow.

### Please provide a concise description of the feature that would resolve your issue.

I would love to be able to have a Markdown mode for creating recipes (similar to the JSON mode). And in cases where a recipe is created this way, it would display the rendered Markdown page in place of "Ingredients", "Steps", and "Note".

### Please consider and list out some caveats or tradeoffs made in your design decision

Recipes created this way would sacrifices some features like being able to add ingredients to the grocery list or search by ingredients for a simpler cookbook experience though special parsing and indexing can be added down the line.

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by hay-kot** [comment]
Hey! I appreciate the ask here, but I think it falls too far outside of what we're already doing. The JSON input is literally just the JSON representation of the existing structure, so that was pretty simple to implement.

To add markdown mode we'd need to implement an entirely new storage, search, tagging, etc... mechanism to work in both cases. I think at that point you're better served by another app instead of trying to fit that feature into Mealie.

**Reply by benjamonnguyen** [reply]
Thank you for the thoughtful response!
